website/frontend/src/helpers.ts

```diff
@@ -28,3 +28,12 @@ export function readFileAsync(file: File){
     reader.readAsArrayBuffer(file);
   });
 };
+
+
+export default function toShortAddress(_address: string) {
```

Comment:
better way to use exports is named,
instead 
```
export default function toShortAddress
//
import toShortAddress from '../utils'
```
use
```
export function toShortAddress
//
import { toShortAddress } from '../utils'
```

Reply:
done